docs/porting-to-ROS2.md

```diff
@@ -111,6 +111,24 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 ```
 
+#### Linters
+Add only `ament_cmake_cppcheck` and `ament_cmake_cpplint` to the list of linters in `package.xml`
+
+```xml
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_cmake_cppcheck</test_depend>
+  <test_depend>ament_cmake_cpplint</test_depend>
```

Comment:
Maybe I missed some discussion but I thought that we would just be doing cppcheck and clang-tidy?

Reply:
I must have misheard @dejanpan, I thought he said he, @nnmm and @fred-apex-ai agreed to use cpplint as well.

Reply:
Fixed in e7a8d65

Reply:
Thanks for clarifying!